mongo_connector/command_helper.py

```diff
@@ -16,59 +16,29 @@
 """
 
 import logging
-import mongo_connector.errors
+from mongo_connector.dest_mapping import DestMapping
 
 LOG = logging.getLogger(__name__)
 
 
 class CommandHelper(object):
-    def __init__(self, namespace_set=[], dest_mapping={}):
-        self.namespace_set = namespace_set
-        self.dest_mapping = dest_mapping
-
-        # Create a db to db mapping from the namespace mapping.
-        db_pairs = set((ns.split('.')[0],
-                        self.map_namespace(ns).split('.')[0])
-                       for ns in self.namespace_set)
-        targets = set()
-        for _, dest in db_pairs:
-            if dest in targets:
-                dbs = [src2 for src2, dest2 in db_pairs
-                       if dest == dest2]
-                raise mongo_connector.errors.MongoConnectorError(
-                    "Database mapping is not one-to-one."
-                    " %s %s have collections mapped to %s"
-                    % (", ".join(dbs),
-                       "both" if len(dbs) == 2 else "all",
-                       dest))
-            else:
-                targets.add(dest)
-
-        self.db_mapping = {}
-        for src, dest in db_pairs:
-            arr = self.db_mapping.get(src, [])
-            arr.append(dest)
-            self.db_mapping[src] = arr
+    def __init__(self, dest_mapping_stru=DestMapping([],[],{})):
```

Comment:
Because in the test_commands of TestSolrDocManager class, there is one case that initiate CommandHelper with no argument and the old __init__ function in CommandHelper has default values as

`def __init__(self, namespace_set=[], dest_mapping={})`

so I changed accordingly, but you are right, this is not necessary and in that testcase, actually we can init a DestMapping object with empty values and then pass to the CommandHelper. I will remove this default value. Also in the DestMapping init.